lib/schema.js

```diff
@@ -38,7 +38,8 @@ var express = require("express"),
     RemoteAccessToken = require("./model/remoteaccesstoken").RemoteAccessToken,
     RemoteRequestToken = require("./model/remoterequesttoken").RemoteRequestToken,
     ActivityObject = require("./model/activityobject").ActivityObject,
-    DatabankStore = require('connect-databank')(express),
+    session = require("express-session"),
+    DatabankStore = require('connect-databank')(session),
```

Comment:
I needed a change in DatabankStore (connect-databank), a close function to stop the clean-up timer. 

I noticed the Migrating guide (https://github.com/strongloop/express/wiki/Migrating-from-3.x-to-4.x), so I thought why not make the connect-databank to be express 4 ready.  Maybe a bit soon, and I still need make a proper pull request for it.

Thanks, by the way, for reviewing.

I'll try to configure jshint to detect the spacing and quoting, I'm a noob in using jshint, don't know if it is possible. I'll try make the changes this weekend, and commit/squash them with the existing commits.